ImageClassification/CIFAR10/DeepResNet/test.sh

```diff
@@ -23,14 +23,14 @@ WEIGHTS_FILE=${WEIGHTS_URL##*/}
 echo "Downloading weights file from ${WEIGHTS_URL}"
 curl -o $WEIGHTS_FILE $WEIGHTS_URL 2> /dev/null
 
-python -u $TEST_SCRIPT  -i ${EXECUTOR_NUMBER} -vvv --model_file $WEIGHTS_FILE --no_progress_bar -w /usr/local/data/CIFAR10/macrobatches | tee output.dat 2>&1
+python -u $TEST_SCRIPT  -i ${EXECUTOR_NUMBER} -vvv --model_file $WEIGHTS_FILE --manifest val:/data/CIFAR/val-index.csv --manifest_root /data/CIFAR -b gpu -z 32 --no_progress_bar 2>&1 | tee output.dat
```

Comment:
can you use "/dataset/aeon/CIFAR10" instead of "/data/CIFAR"?

Reply:
@baojun-nervana Out of curiosity, is there any particular reason why these paths should be changed?

Reply:
@tpatejko that is the validation dataset directory, so we don't have to update those after the PR.

Reply:
Changed